.devcontainer/devcontainer.json

```diff
@@ -1,31 +1,27 @@
 {
     "name": "Azure Developer CLI",
-    "image": "mcr.microsoft.com/devcontainers/javascript-node:18-bookworm",
+    "image": "mcr.microsoft.com/devcontainers/typescript-node:dev-18-bullseye",
     "features": {
+        "ghcr.io/devcontainers/features/docker-in-docker:2": {},
         "ghcr.io/azure/azure-dev/azd:latest": {},
-        "ghcr.io/devcontainers/features/azure-cli:latest": {},
-        "ghcr.io/devcontainers/features/docker-in-docker:latest": {},
-        "ghcr.io/devcontainers/features/github-cli:latest": {},
-        "ghcr.io/devcontainers/features/powershell:latest": {}
+        "ghcr.io/devcontainers-contrib/features/npm-package:1": {},
+        "ghcr.io/devcontainers/features/azure-cli:1": {}
```

Comment:
We should use `:latest` for each feature, instead of pinning the version.

Reply:
It's confusing, but that version is the version of the devcontainer feature, not the version of the CLI.  Here's more info: https://github.com/devcontainers/features/tree/main/src/azure-cli

The Azure CLI version is an option of the feature, which defaults to latest, so is omitted in the devcontainer.json file.

Reply:
No, it's not confusing.

Azure CLI for example, https://github.com/devcontainers/features/pkgs/container/features%2Fazure-cli, we just take the `latest` tag like `"ghcr.io/devcontainers/features/azure-cli:latest": {}`, instead of `"ghcr.io/devcontainers/features/azure-cli:1": {}`

Reply:
ic, ok, I didn't realize that supported 'latest' as well.